starter-code/index.html

```diff
@@ -1,31 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <title>Instagram</title>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" media="screen" href="./css/main.css" />
+
 </head>
+
 <body>
-  Instagram
 
-  Capture and Share the World's Moments
+  <section>
+
+    <div id="phones">
+      <img src="images/phones.png">
+    </div>
+
+    <div id="content">
+
+      <header>
+        
+        <a href =""><img src="./images/brand.png" style="width: 80%"></a>
```

Comment:
in your css, you should make the a's width and height set to the image's width and height. 

Because your A tag is bigger than the image and it's not the best if you want to align items.